docs/manual/meta/application_space_warp.rst

```diff
@@ -0,0 +1,20 @@
+Meta Application Space Warp
+===========================
+
+.. note::
+
+    Check out the `Meta Space Warp Sample Project <https://github.com/GodotVR/godot_openxr_vendors/tree/master/samples/meta-space-warp-sample>`_
+    for a working demo of Application Space Warp.
+
+Application Space Warp is an extension for Meta XR devices that allows applications to render at half FPS while the runtime
+generates the in-between frames, perserving a smooth experience for users while lowering the workload of the device's CPU/GPU.
```

Comment:
Maybe extend this to say something about using the extra GPU power to generate better visuals?